arch/xtensa/src/esp32/esp32_spi.c

```diff
@@ -158,6 +190,20 @@ static void esp32_spi_deinit(FAR struct spi_dev_s *dev);
  * Public Function Prototypes
  ****************************************************************************/
 
+extern uint8_t esp32_spi2_status(FAR struct spi_dev_s *dev, uint32_t devid);
+extern uint8_t esp32_spi3_status(FAR struct spi_dev_s *dev, uint32_t devid);
+
+/* Use board level CS select function */
+
+#ifdef CONFIG_ESP32_SPI_UDCS
+extern void esp32_spi2_select(FAR struct spi_dev_s *dev,
+                              uint32_t devid,
+                              bool selected);
+extern void esp32_spi3_select(FAR struct spi_dev_s *dev,
+                              uint32_t devid,
+                              bool selected);
+#endif
+
```

Comment:
These are already prototyped in the header file.

Reply:
I will remove these including `esp32_spi2_status` and `esp32_spi3_status`, I see they all prototyped in the header file.